db/recovery_db.go

```diff
@@ -109,6 +138,23 @@ func (db *RecoveryDB) PutSwapPrivateKey(id types.Hash, keys *mcrypto.PrivateKeyP
 	return db.db.Put(key[:], val)
 }
 
+// GetSwapPrivateKey returns the swap private key share, if it exists.
```

Comment:
"Private key share" is vague given all the keys our code deals with. I assume this is the private spend key of the Monero A+B wallet? Do we want to adopt some kind of consistent terminology for the A+B wallet that we use in the code everywhere?

Reply:
Looking further down, I think the next function is returning what I thought this function might be returning.  Would it be correct to say that this one is returning "Alice or Bob's half of the A+B wallet spend key"?

Reply:
key share generally means one part of a private key, which in this case means one party's "half" of the A+B key.  the "shared" key is the private key of the A+B wallet (a different function)

I've been using "shared" key for the A+B wallet, do you have a preference for clearer naming?

Reply:
just saw your second comment, yes exactly

Reply:
So let's put the word "Share" in this function. `GetSwapPrivateKeyShare`? Right now the comment says "the swap private key share" implying that there is only one, but there are 2 and this key is our half if I understand correctly.  The next function is named `GetSharedSwapPrivateKey`. Maybe we can remove the word "Shared" from it so no one can confuse it with the word "Share". I don't have a suggestion, as I still don't know what it is storing without digging through lots of code. Is it storing the private view key on Bob's side and the private spend key on Alice's side?

Reply:
I think `GetSharedSwapPrivateKey` is fine as the key is essentially a Diffie-Hellman shared key. the only other term that might be okay is "common key"? 

I could also update the comment to say "our swap private key share" to make it clearer?

> Is it storing the private view key on Bob's side and the private spend key on Alice's side?
It's storing the private (spend and view) key on both sides. it corresponds to `swapState.privkeys` for both